uint8 TABLE_TOP=1
uint8 SURFACE=2
uint8 type
string name
geometry_msgs/Pose pose
geometry_msgs/Vector3 dimensions